src/jwplayer/players/DesktopArticleVideoPlayer/DesktopArticleVideoPlayer.tsx

```diff
@@ -50,25 +58,43 @@ export const DesktopArticleVideoPlayerContent: React.FC<DesktopArticleVideoPlaye
 							isScrollPlayer ? styles.desktopArticleVideoWrapperScrollPlayer : styles.desktopArticleVideoWrapper
 						}
 					>
-						{adComplete && (
+						{jwpAdsSetupComplete.strategyRulesEnabled ? (
 							<div>
 								<div className={styles.topBar}>
 									{!isScrollPlayer && <UnmuteButton />}
 									{isScrollPlayer && (
 										<CloseButton dismiss={() => setDismissed(true)} iconColor={'#fff'} className={styles.closeButton} />
 									)}
 								</div>
-								<JwPlayerWrapper
+								<StrategyRulesWrapper
 									getDismissed={getDismissed}
 									config={getArticleVideoConfig(videoDetails)}
-									onReady={(playerInstance) => {
-										articlePlayerOnReady(videoDetails, playerInstance);
-										setIsPlayerReady(true);
-									}}
-									stopAutoAdvanceOnExitViewport={false}
+									onReady={onPlayerInstanceReady}
 								/>
 								<input type="hidden" value={String(dismissed)} name={inputName} />
 							</div>
+						) : (
+							adEngineComplete && (
+								<div>
+									<div className={styles.topBar}>
+										{!isScrollPlayer && <UnmuteButton />}
+										{isScrollPlayer && (
+											<CloseButton
+												dismiss={() => setDismissed(true)}
+												iconColor={'#fff'}
+												className={styles.closeButton}
+											/>
```

Comment:
do we need to have this code duplicated? 🤔

Reply:
👍 I even started writing a `DesktopArticleVideoFrame` component ;) But then again once we decide if we want to go with strategy rules globally or not it all will be unnecessary 😊